pkg/goformatters/gci/internal/config/config.go

```diff
@@ -0,0 +1,106 @@
+package config
+
+import (
+	"sort"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/daixiang0/gci/pkg/config"
+	"github.com/daixiang0/gci/pkg/section"
```

Comment:
I think the problem here.

"github.com/daixiang0/gci/pkg/section"
vs
"github.com/golangci/golangci-lint/pkg/goformatters/gci/internal/section"

Reply:
I don't so because `Section` is an interface

Reply:
you created the package `github.com/golangci/golangci-lint/pkg/goformatters/gci/internal/section`
but the programme won't get into it.

I went through this in debugging

Reply:
The problem is related to the `Parse`, I will fix that.

Reply:
Thank you for the quick fix

Reply:
https://github.com/golangci/golangci-lint/pull/5407